### ISSUE TITLE
Latin characters accentuation

### ISSUE THREAD

**Issue by marcocspc** [terrychou/iVim#83]
Hello! I’m having problems with latin accented characters. When I type the character is shown normally, but with its accent’s characters on its side. Like this: é’é’, ã~, and so on... I’m attaching a screen capture to show it on the app.

![IMG_0090](https://user-images.githubusercontent.com/17876596/71369078-69db9680-2588-11ea-8cfd-237a93ae65c5.png)

**Comment by terrychou** [comment]
Please provide the way to reproduce the problem, for example, how do I input the accented characters? And the versions of your app and system.

**Comment by marcocspc** [comment]
Thanks for the quick response.

To type the accented characters you should use apple magic keyboard, international version. On System Settings App > General > Keyboard > Physical Keyboard, you should select International USA - PC for input language. Then you can type pressing tilde (under esc key) and A or O, another way would be pressing single quote (‘) and then A, E, I, O or U.

Software version: iVim 2.10 and iPadOS 13.1.3

**Comment by terrychou** [comment]
Unfortunately, I don’t have a magic keyboard by hand. I will see whether I can test it in other ways.

**Comment by terrychou** [comment]
It's been fixed. The fix will be available in the upcoming update.

**Comment by AmaruCoder** [comment]
I have been dealing with this issue too, and have been waiting for the next release. Is there any way to get early access?

**Comment by AmaruCoder** [comment]
@terrychou thank you very much for the quick turnaround. I have installed the new version, and found a strange behavior. I am using an external keyboard for these tests. For comparison, this is what happened in different versions:

**Version 2.00  (and all other editors in iOS)**
`<alt-e> + e = é` (the expected behavior)

**Version 2.10**
`<alt-e> + e = é´` (notice the accent mark after the letter)

**Version 2.12**

Alt enabled:
`<alt-e> + e = åe`

Alt disabled:

`<alt-e> + e = e`
`<alt-e> + e + <backspace> =` crashes iVim

I wonder if version 2.10 has change the keyboard handler, and, if so, it could be restored to the one used in 2.00.

**Comment by terrychou** [comment]
@AmaruCoder this is caused by the conflict between this and the newly added `Key Repeating` option (#88). For now, you need to switch the latter off (it is on by default) to make the former work. I will find a fix.

**Comment by terrychou** [comment]
@AmaruCoder and it's been fixed. If you would like to test it, please give me an email.

**Comment by AmaruCoder** [comment]
Great. Thank you very much for looking into it. I would like to test the new version. My email is `escribidor` at `att.net`.

**Comment by AmaruCoder** [comment]
Thank you for the invitation to test iVim. Just one question, would installing the test version of iVim delete the data in the app?

**Comment by terrychou** [comment]
@AmaruCoder although it warned me of this sometimes, I don't remember any data loss. However, you better test it with a safer settings, backup you data first.

**Comment by AmaruCoder** [comment]
Thank you. The new version works great. However, there is a new issue. When I switch, using an external keyboard, and with <command-tab> to a new application, sometimes iVim look like the enclosed image (notice the black band above and the compressed characters). This happened more often with PDF Expert (a PDF viewer). I imagine that this has to do with the new Italic and Boldface support.

![Issue](http://josedepierola.com/media/ivim-screen.PNG)

**Comment by terrychou** [comment]
It should have nothing to do with the bold and italic style support.

Do you have a way to reproduce it?

**Comment by AmaruCoder** [comment]
Yes, you open iVim, then `<command-tab>` to PDF Expert and back to iVim. Perhaps PDF expert sets up properties for the screen that iVim doesn't clear out? This is something that didn't happen in the previous version of iVim.

**Comment by terrychou** [comment]
@AmaruCoder I tried with PDF Expert 7 but failed to reproduce this problem.

**Comment by AmaruCoder** [comment]
Thank you very much for the help. Unfortunately, I cannot reproduce the issue consistently. It happens sometimes but not in a way I can reproduce. Thank you again for the time, and for making Vim available on the iPad. It has changed the way I work.
